### PR TITLE
QAART-821 Add groups for Mercury related features

### PR DESCRIPTION
**Pull request by nikodamn**
Reviewers:
@ludwikkazmierczak

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java [hunk 1]
```diff
@@ -109,7 +109,7 @@ public void MercuryLoginTest_009_registerButtonWorksOnJoinPage() {
     Assertion.assertEquals(registrationPage.getRegisterHeaderText(), expectedHeader);
   }
 
-  @Test(groups = "MercuryLoginTest_010")
+  @Test(groups = {"MercuryLoginTest_010", "LoginTests_002"})
   public void MercuryLoginTest_010_signInLinkWorksOnJoinPage() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java [hunk 2]
```diff
@@ -129,7 +129,7 @@ public void MercuryLoginTest_011_japaneseUserLogIn() {
 //        Configuration.getCredentials().userNameJapanese2));
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java [hunk 3]
```diff
@@ -129,7 +129,7 @@ public void MercuryLoginTest_011_japaneseUserLogIn() {
...
   }
 
-  @Test(groups = "MercuryLoginTest_012")
+  @Test(groups = {"MercuryLoginTest_012", "LoginTests_002"})
   public void MercuryLoginTest_012_passwordTogglerWorks() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java [hunk 4]
```diff
@@ -97,7 +97,7 @@ public void MercuryLoginTest_008_userIsTakenToJoinPage() {
     Assertion.assertEquals(joinPageObject.getJoinTodayText(), expectedMessage);
   }
 
-  @Test(groups = "MercuryLoginTest_009")
+  @Test(groups = {"MercuryLoginTest_009", "LoginTests_002"})
   public void MercuryLoginTest_009_registerButtonWorksOnJoinPage() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java [hunk 5]
```diff
@@ -27,39 +27,39 @@
...
   public void MercuryLoginTest_004_notPossibleToLogInWhenUsernameFieldBlank() {
     LoginPageObject loginPageObject = new LoginPageObject(driver).get();
     loginPageObject.logUserIn("", Configuration.getCredentials().password10);
 
     Assertion.assertTrue(loginPageObject.isSubmitButtonDisabled());
   }
 
-  @Test(groups = "MercuryLoginTest_005")
+  @Test(groups = {"MercuryLoginTest_005", "LoginTests_001"})
   public void MercuryLoginTest_005_notPossibleToLogInWhenPasswordFieldBlank() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java [hunk 6]
```diff
@@ -27,39 +27,39 @@
...
   public void MercuryLoginTest_002_userCanNotLogInWithWrongPassword() {
     LoginPageObject loginPageObject = new LoginPageObject(driver).get();
     loginPageObject.logUserIn(Configuration.getCredentials().userName10, "thisIsWrongPassword");
 
     Assertion.assertEquals(loginPageObject.getErrorMessage(), ERROR_MESSAGE);
   }
 
-  @Test(groups = "MercuryLoginTest_003")
+  @Test(groups = {"MercuryLoginTest_003", "LoginTests_001"})
   public void MercuryLoginTest_003_invalidUserCanNotLogIn() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java [hunk 7]
```diff
@@ -84,7 +84,7 @@ public void MercuryLoginTest_007_registerNowLinkWorks() {
     Assertion.assertEquals(expectedHeader, currentHeader);
   }
 
-  @Test(groups = "MercuryLoginTest_008")
+  @Test(groups = {"MercuryLoginTest_008", "LoginTests_002"})
   public void MercuryLoginTest_008_userIsTakenToJoinPage() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java [hunk 8]
```diff
@@ -120,7 +120,7 @@ public void MercuryLoginTest_010_signInLinkWorksOnJoinPage() {
     Assertion.assertEquals(loginPageObject.getLoginHeaderText(), expectedHeader);
   }
 
-  @Test(groups = "MercuryLoginTest_011")
+  @Test(groups = {"MercuryLoginTest_011", "LoginTests_002"})
   public void MercuryLoginTest_011_japaneseUserLogIn() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java [hunk 9]
```diff
@@ -72,7 +72,7 @@ public void MercuryLoginTest_006_closeButtonWorksAndRedirectsProperly() {
     Assertion.assertEquals(expectedHomePageTitle, homePage.getArticleTitle());
   }
 
-  @Test(groups = "MercuryLoginTest_007")
+  @Test(groups = {"MercuryLoginTest_007", "LoginTests_002"})
   public void MercuryLoginTest_007_registerNowLinkWorks() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java [hunk 10]
```diff
@@ -27,39 +27,39 @@
...
   public void MercuryLoginTest_005_notPossibleToLogInWhenPasswordFieldBlank() {
     LoginPageObject loginPageObject = new LoginPageObject(driver).get();
     loginPageObject.logUserIn(Configuration.getCredentials().userName10, "");
 
     Assertion.assertTrue(loginPageObject.isSubmitButtonDisabled());
   }
 
-  @Test(groups = "MercuryLoginTest_006")
+  @Test(groups = {"MercuryLoginTest_006", "LoginTests_001"})
   public void MercuryLoginTest_006_closeButtonWorksAndRedirectsProperly() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java [hunk 11]
```diff
@@ -27,39 +27,39 @@
...
   public void MercuryLoginTest_003_invalidUserCanNotLogIn() {
     LoginPageObject loginPageObject = new LoginPageObject(driver).get();
     loginPageObject.logUserIn("notExistingUserName", Configuration.getCredentials().password10);
 
     Assertion.assertEquals(loginPageObject.getErrorMessage(), ERROR_MESSAGE);
   }
 
-  @Test(groups = "MercuryLoginTest_004")
+  @Test(groups = {"MercuryLoginTest_004", "LoginTests_001"})
   public void MercuryLoginTest_004_notPossibleToLogInWhenUsernameFieldBlank() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java [hunk 12]
```diff
@@ -27,39 +27,39 @@
   private static final String ERROR_MESSAGE =
       "We don't recognize these credentials. Try again or register a new account.";
 
-  @Test(groups = "MercuryLoginTest_002")
+  @Test(groups = {"MercuryLoginTest_002", "LoginTests_001"})
   public void MercuryLoginTest_002_userCanNotLogInWithWrongPassword() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 12 issues:
* MAJOR 12 major

Watch the comments in this conversation to review them.

**Comment by ludwikkazmierczak** [comment]
LGTM

**Pull request merged**